contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+
+A relatively new tool called <a target="_blank" rel="noopener noreferrer nofollow" href="http://pry.co/">Pry</a> has been a game-changer for us. Yes, we all know you can “just” do it in Excel (which you will want to if you have a finance background), but a tool like Pry makes it so much easier. And you can skip the 2 years chained to your desk at Goldman!
+
+Pry connects seamlessly with your accounting software, and you then just fill out assumptions manually or using very simple formulae. Pry also has neat dashboards for visualising everything, as well as scenario modelling. They currently charge $100/month, though worth asking for a discount if you are a startup...
+
+## Where to put the money
+
+<iframe src="https://giphy.com/embed/3o6ZtpoNGDS4cZ8GWY" width="480" height="338" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/southparkgifs-3o6ZtpoNGDS4cZ8GWY">via GIPHY</a></p>
+
+Put it in <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.brex.com/">Brex</a>, obviously (US only). They also make it incredibly easy to manage virtual credit cards, and their built-in expense management negates the need for something like Expensify. They don’t charge you anything.
+
+In Europe, <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.revolut.com/">Revolut</a> are about the only digital bank that get anywhere near Brex features-wise, although I find their platform a lot more buggy.
+
+## Ok, but what about…?
+
+Recruitment? Sales? HR? Business planning? You may or may not consider these ops, but for the sake of simplicity I’m throwing them in under ‘anything that is not building the product or talking to customers’.
+
+You can do these perfectly fine in a well-organized Google Doc or Sheet. Just be disciplined and keep it up to date. Once you start spending half a day per week on these things, then you should invest in:
+
+- <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.workable.com/">Workable</a> for recruitment. $99 per role per month, pay as you go.
+- <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.hubspot.com/">HubSpot</a> for sales. Various tiers and combos available, starting from free. Free doesn’t mean you should start there though.
+- Something like <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.charliehr.com/">Charlie</a> or <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.hibob.com/">Bob</a> for things like recording salaries, addresses etc.. Around $6 per employee per month usually.
+
+For business planning, I’m yet to find a tool that genuinely convinces me of its value over a well-organized Google Sheet. The number one challenge I’ve found is effectively communicating and tracking the output of business planning, and people really don’t like having to use another multiple platforms to track what they’re doing.
```

Comment:
What's difference between business planning + pry in your mind?

Reply:
Business planning to me is setting objectives like OKRs and tracking them, whereas Pry is financial planning only.